.github/workflows/update-cache.yaml

```diff
@@ -98,27 +100,38 @@ jobs:
           token: ${{ secrets.SANDPAPER_WORKFLOW }}
           delete-branch: true
           branch: "update/packages"
-          commit-message: "[actions] update ${{ steps.update.outputs.n }} packages"
+          commit-message: >
+            [actions] update ${{ steps.update.outputs.n }} packages
           title: "Update ${{ steps.update.outputs.n }} packages"
-          body: |
+          body: >
             :robot: This is an automated build
 
-            This will update ${{ steps.update.outputs.n }} packages in your lesson with the following versions:
+            This will update ${{ steps.update.outputs.n }} packages
+            in your lesson with the following versions:
+
 
             ```
+
             ${{ steps.update.outputs.report }}
+
             ```
```

Comment:
I worry about this change because `${{ steps.update.outputs.report }}` is a mutli-line variable (see example in https://github.com/carpentries/sandpaper-docs/pull/132) and it would collapse into a single line if I left it alone or, if I [the function that produces the report](https://github.com/zkamvar/vise/blob/ee4798701a958ee48429980eb970266885f8265b/R/update.R#L97-L109), then outdated workflows would have a doubling of the number of lines.